src/main/java/urjc/isi/dao/implementaciones/PeliculasDAOImpl.java

```diff
@@ -128,9 +129,25 @@ public void deleteByID(String idpelicula){
 					     "Inner join actores as a on pa.idpersona=a.idpersona ";
 					cond+= "a.fullnombre LIKE "+"'"+conditions.get("actor")+"'";
 					break;
+				case "director":
+					sql+="Inner join peliculasdirectores as pd on p.idpelicula=pd.idpelicula " +
+						 "Inner join directores as d on pd.idpersona=d.idpersona ";
+					cond+= "d.fullnombre LIKE "+"'"+conditions.get("director")+"'";
+					break;
+				case "guionista":
+					sql+="Inner join peliculasguionistas as pg on p.idpelicula=pg.idpelicula " +
+						 "Inner join guionistas as g on pg.idpersona=g.idpersona ";
+					cond+= "g.fullnombre LIKE "+"'"+conditions.get("guionista")+"'";
+					break;
 				case "duracion":
 					cond+= "p.duracion>"+"'"+conditions.get("duracion")+"'";
 					break;
+				case "adultos":
+					cond+= "calificacion::INTEGER = 1";
+					break;
+				case "ninos":
+					cond+= "calificacion::INTEGER = 0";
+					break;
```

Comment:
Estas dos las podríamos dejar reducidas a una. ?Adultas=no, pues solo imprime las que tienen el 0, y si no pone nada, pues por defecto te imprime todas. ¿Qué os parece? Más que nada porque al depender las dos de la misma columna, creo que deberían ser las dos la misma query del servidor.

Reply:
En el caso de que no todas las peliculas fuesen para "niños", no tendría mas sentido dejar las dos?? peliculas/selectAll?adultos y peliculas/selectAll?ninos

Reply:
Normalmente lo interesante es hacer que las pelis de adultos no aparezcan, pero que aparezcan solo las de adultos puede ser también una posibilidad. Para ello es mejor jugar con una sola variable aun así. ?adultos=si/no/solamente como queráis hacerlo. El mayor inconveniente es lo de trabajarlo con dos variables separadas cuando es algo que llama a la misma columna de la tabla. Pero vamos este es mi razonamiento nada más